server/polar/models/transaction.py

```diff
@@ -240,6 +240,12 @@ class Transaction(RecordModel):
     fee_balance_transaction_id: Mapped[str | None] = mapped_column(
         String, nullable=True, index=True
     )
+
+    """Payment risk level."""
+    risk_level: Mapped[str | None] = mapped_column(String, nullable=True, index=False)
+    """Payment risk score (0-100) for payments."""
+    risk_score: Mapped[str | None] = mapped_column(Integer, nullable=True, index=False)
```

Comment:
Type hint should be `Mapped[int | None]`

Reply:
Doh! Silly copy-paste mistake 😬